types/redux-first-router/index.d.ts

```diff
@@ -152,9 +183,15 @@ export interface Options {
     onAfterChange?<S>(dispatch: Dispatch<S>, getState: StateGetter): void;
     onBackNext?<S>(dispatch: Dispatch<S>, getState: StateGetter): void;
     restoreScroll?(history: History): ScrollBehavior;
+    initialDispatch?: boolean;
+    querySerializer?: QuerySerializer;
     navigators?: NavigatorsConfig;
 }
 
+// Params and Payload differ from Flow definition, as they can be arbitrarily shaped
+export type Params = any;
+export type Payload = any;
+
```

Comment:
pardon my ignorance - what is the correct way to do things? Leaving `Params` and `Payload ` as `object` doesn't work because both can have arbitrary shape, i.e. you'd be able to access `params.slug` in one use case and `params.id` in another (I think the `object` typing was copied from the Flow type definitions). Still new to TypeScript, so I am most likely totally missing the mark by using `any`.